src/libraries/System.Collections/ref/System.Collections.cs

```diff
@@ -580,6 +580,8 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<T>, Sy
         }
     }
 }
+#endif // !BUILDING_CORELIB_REFERENCE
+
 namespace System.Collections.ObjectModel
 {
     public partial class ReadOnlySet<T> : System.Collections.Generic.ICollection<T>, System.Collections.Generic.IEnumerable<T>, System.Collections.Generic.IReadOnlyCollection<T>, System.Collections.Generic.IReadOnlySet<T>, System.Collections.Generic.ISet<T>, System.Collections.ICollection, System.Collections.IEnumerable
```

Comment:
Seeing as the other `System.Collections.ObjectModel` collections that are already in System.Private.CoreLib are not listed in this file, does `ReadOnlySet<T>` still need to be here?

Reply:
When I remove it here, I get this error when running `build.cmd clr+libs -rc Release`:
```
C:\Code\Other\dotnet\runtime\src\libraries\System.Collections\ref\System.Collections.cs(600,54): error CS0234: The type
 or namespace name 'ReadOnlySet<>' does not exist in the namespace 'System.Collections.ObjectModel' (are you missing an
 assembly reference?) [C:\Code\Other\dotnet\runtime\src\libraries\System.Private.CoreLib\ref\System.Private.CoreLib.csp
roj]
```

Reply:
It should be added to the System.Runtime reference project:

https://github.com/dotnet/runtime/blob/07f984305205a760edd2ace52670eefdaf3072a2/src/libraries/System.Runtime/ref/System.Runtime.cs#L8367-L8399

Here are instructions on how to regenerate the reference file:

https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md

Reply:
I had no luck with these instructions as builds failed as soon as I removed it from the System.Collections ref project. So I added it manually to System.Runtime ref project.